test/src/test/java/jenkins/model/JenkinsBuildsAndWorkspacesDirectoriesTest.java

```diff
@@ -66,6 +69,31 @@ private void clearSystemProperties() {
         Stream.of(Jenkins.BUILDS_DIR_PROP, Jenkins.WORKSPACES_DIR_PROP)
                 .forEach(System::clearProperty);
     }
+    
+	@Issue("JENKINS-53284")
+	@Test
+	public void changeWorkspacesDirLog() throws Exception {
+		loggerRule.record(Jenkins.class, Level.WARNING)
+				.record(Jenkins.class, Level.INFO).capture(1000);
+
+		story.then(step -> {
+			assertFalse(logWasFound(LOG_WHEN_CHANGING_WORKSPACES_DIR));
+			setWorkspacesDirProperty("testdir1");
+		});
+
+		story.then(step -> {
+			assertTrue(logWasFoundAtLevel(LOG_WHEN_CHANGING_WORKSPACES_DIR,
+					Level.WARNING));
+			setWorkspacesDirProperty("testdir2");
+		});
+
+		story.then(step -> {
+			assertTrue(logWasFoundAtLevel(LOG_WHEN_CHANGING_WORKSPACES_DIR,
+					Level.WARNING));
+		});
+
+	}
+    
```

Comment:
Your code formatter seems to have an issue :-)
![image](https://user-images.githubusercontent.com/223853/51172485-9df80300-18b3-11e9-9e30-4ed248a2660a.png)